### ISSUE TITLE
Various typos and points of confusion in chapter 2

### ISSUE THREAD

**Issue by willwerscheid** [mdsr-book/mdsr-book.github.io#44]
Sorry if it is annoying to submit multiple issues this way --- please let me know if you would prefer individual tickets for each issue. Really love this book --- I am using it as the main textbook for my intro to data science course this semester so more issues will probably be on the way....

2.1: 
using commas for decimal notation could be confusing to an American reader:
> in particular, _$309, 0_ million of the _$337, 8_ million spent on President Obama’s campaign
> the percentage of negative spending on Romney’s campaign (_56, 94%_) was lower than that of Obama (_92, 1%_)

2.1.1
I don't think Figure 2.4 supports this assertion?
> it was only in the presidential election that more money was spent attacking Democratic candidates. In fact, _slightly more money_ was spent attacking Republican House and Senate candidates.

2.1.2
Figure 2.6 is missing a panel (only shows primary spending, not primary vs general)
> [Figure 2.6](https://mdsr-book.github.io/mdsr3e/02-datavizI.html#fig-donations-split) allows us to make more nuanced conclusions. _The right panel_ supports...

2.1.4
Should Romney and Brown be labelled in Figure 2.9? This sentence is hard to follow otherwise:
> In [Figure 2.9](https://mdsr-book.github.io/mdsr3e/02-datavizI.html#fig-ma-funding), we see that the two campaigns that benefited the most from committee spending were Republicans [Mitt Romney](https://en.wikipedia.org/w/index.php?search=Mitt%20Romney) and [Scott Brown](https://en.wikipedia.org/w/index.php?search=Scott%20Brown).

2.2.3
Consider rewriting:
> On the right, we see the distribution of substance for housed (no nights in shelter or on the street) participants is fairly evenly distributed, while on the left, we see the same distribution for those who were homeless one or more nights (more likely to have alcohol as their primary substance of abuse).

Maybe something like this instead: 
> On the right, we see that the distribution of substance for housed participants (no nights in shelter or on the street) is fairly evenly distributed, while the left chart shows that participants who were homeless one or more nights are more likely to have alcohol as their primary substance of abuse.

2.5
Labels in Figure 2.20 are duplicated for some reason (e.g., "(a) (a)")

**Comment by nicholasjhorton** [comment]
Many thanks for the multiple helpful edits and suggestions. I will work to address them over the coming weeks.

Please keep those cards and letters coming!

**Comment by beanumber** [comment]
These have been addressed, except for the comment about [Figure 2.4](https://mdsr-book.github.io/mdsr3e/02-datavizI.html#fig-raised-office). It's confusing, but it seems OK to me. Am I wrong?

**Comment by willwerscheid** [comment]
In all three cases (H, P, and S) the red portion (money spent attacking) is larger (both in magnitude and as a proportion) for democrats than for republicans. So I think it is not true that

> more money was spent attacking Republican House and Senate candidates

**Comment by beanumber** [comment]
OK, I see.
